src/app/containers/ChartbeatAnalytics/index.test.jsx

```diff
@@ -196,4 +128,36 @@ describe('Charbeats Analytics Container', () => {
 
     expect(tree).toMatchSnapshot();
   });
+
+  it('should call sendCanonicalChartbeatBeacon when platform is canonical, and toggle enabled for chartbeat on local', () => {
+    const expectedConfig = {
+      uid: 50924,
+      domain: 'test-domain',
+      idSync: {
+        bbc_hid: 'cookie',
+      },
+      path: '/',
+      sections: 'secction1 section2',
+      title: 'This is a canonical page article',
+      type: 'article',
+      useCanonical: true,
+      virtualReferrer: '/some-path',
+    };
+
+    const mockGetConfig = jest.fn().mockReturnValue(expectedConfig);
+    testUtils.getConfig = mockGetConfig;
+    mount(
+      <ContextWrap
+        platform="canonical"
+        pageType="article"
+        origin={localBaseUrl}
+      >
+        <ChartbeatAnalytics data={frontPageData} />
+      </ContextWrap>,
+    );
+
+    expect(sendCanonicalChartbeatBeacon).toHaveBeenCalledTimes(1);
+    expect(sendCanonicalChartbeatBeacon).toHaveBeenCalledWith(expectedConfig);
+    expect(testUtils.getConfig).toHaveBeenCalledTimes(1);
+  });
```

Comment:
Are we testing that `AmpChartbeatBeacon` is not rendered if the platform is canonical anywhere?

Reply:
@jamesdonoh no we do not, I could add an assertion for that where we're checking that platform is canonical though